src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -1235,10 +1237,29 @@ void CodegenNeuronCppVisitor::print_nrn_jacob() {
     printer->add_newline(2);
     printer->add_line("/** nrn_jacob function */");
 
-    printer->fmt_line(
+    printer->fmt_push_block(
         "static void {}(_nrn_model_sorted_token const& _sorted_token, NrnThread* "
-        "_nt, Memb_list* _ml_arg, int _type) {{}}",
-        method_name(naming::NRN_JACOB_METHOD));
+        "_nt, Memb_list* _ml_arg, int _type)",
+        method_name(naming::NRN_JACOB_METHOD));  // begin function
+
+    printer->add_multi_line(
+        "_nrn_mechanism_cache_range _lmr{_sorted_token, *_nt, *_ml_arg, _type};");
+
+    printer->fmt_line("auto inst = make_instance_{}(_lmr);", info.mod_suffix);
+    printer->fmt_line("auto node_data = make_node_data_{}(*_nt, *_ml_arg);", info.mod_suffix);
+    printer->fmt_line("auto nodecount = _ml_arg->nodecount;");
+    printer->push_block("for (int id = 0; id < nodecount; id++)");  // begin for
+
+    if (breakpoint_exist()) {
+        printer->add_line("// set conductances properly");
+        printer->add_line("int node_id = node_data.nodeindices[id];");
+        printer->fmt_line("node_data.node_diagonal[node_id] += inst.{}[id];",
+                          info.vectorize ? naming::CONDUCTANCE_UNUSED_VARIABLE
+                                         : naming::CONDUCTANCE_VARIABLE);
```

Comment:
Roughly the same thing as we do here: https://github.com/BlueBrain/nmodl/blob/2edf9aca43bdb80fa827b98f03a0d315c1e34e7c/src/codegen/codegen_cpp_visitor.cpp#L964-L975